nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -78,7 +78,7 @@ let cfg = config.system.autoUpgrade; in
           HOME = "/root";
         } // config.networking.proxy.envVars;
 
-      path = [ pkgs.gnutar pkgs.xz.bin config.nix.package.out ];
+      path = [ pkgs.gnutar pkgs.xz.bin pkgs.git config.nix.package.out ];
```

Comment:
How about `gitMinimal` instead, it's about 25% smaller

Reply:
Thanks for the tip